crates/loader/src/local.rs

```diff
@@ -251,6 +259,57 @@ impl LocalLoader {
         file_content_ref(path)
     }
 
+    async fn load_registry_source(
+        &self,
+        registry: Option<&String>,
+        package: &str,
+        version: &semver::Version,
+    ) -> Result<ContentRef> {
+        let package: wasm_pkg_loader::PackageRef = package
+            .parse()
+            .with_context(|| format!("{package} is not a valid registry package reference"))?;
```

Comment:
Since I have nitted once I shall nit again:

```suggestion
            .with_context(|| format!("{package:?} is not a valid registry package reference"))?;
```

(`Debug` in error messages helps highlight surprise whitespace, etc)